crm_phonecall_summary_restrict/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+Restricted Summary for Phone Calls
+==================================
+
+This module extends the functionality of CRM phonecalls to support setting a
+limited list of possible phonecall summaries, that only the sales manager can
+edit.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Go to *Sales > Configuration > Phone Calls > Summaries*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to *Sales > Phone Callse > Logged Calls*.
```

Comment:
Put that "there you will find a new field called x that allows to set the phone call summary".